sdks/go/pkg/beam/artifact/gcsproxy/staging.go

```diff
@@ -18,8 +18,9 @@ package gcsproxy
 
 import (
 	"bytes"
-	"crypto/md5"
+	"crypto/sha256"
 	"encoding/base64"
```

Comment:
Do we still need this?

Reply:
No, we don't need it. 
I must have missed removing it.